ledger/src/builtins.rs

```diff
@@ -1,8 +1,19 @@
-use solana_runtime::builtins::{Builtin, BuiltinFeatureTransition, Builtins};
+use {
+    solana_program_runtime::invoke_context::InvokeContext,
+    solana_runtime::builtins::{
+        process_instruction_with_program_logging, Builtin, BuiltinFeatureTransition, Builtins,
+    },
+};
 
-macro_rules! to_builtin {
+macro_rules! to_builtin_with_logging {
```

Comment:
If all builtins now include logging why th need to distinguish that in the name?